Source/OxyPlot.Avalonia/PlotBase.cs

```diff
@@ -334,8 +340,8 @@ public void ShowTracker(TrackerHitResult trackerHitResult)
             if (!object.ReferenceEquals(tracker, currentTracker))
             {
                 HideTracker();
-                overlays.Children.Add(tracker.Control);
-                currentTracker = tracker.Control;
+                overlays.Children.Add(tracker.Result);
+                currentTracker = tracker.Result;
             }
```

Comment:
Not an issue with the PR, but might be a good opportunity to fix anyway: I think this is a bug dating back to when this was ported from WPF: presumably we should take `tracker = trackerTemplate.Build(new ContentControl());` so that the object comparison can return true.